src/com/xilinx/rapidwright/router/SATRouter.java

```diff
@@ -773,7 +773,7 @@ public void route(){
 		createNetsFiles();
 		createPipFile();
 		int result = runEvRouter();
-		if(result != 0){
+		if(result != 0 && result != 1){
```

Comment:
What does return code of 1 mean?

Reply:
From the author of `evRouter`:
> It might not be intuitive but I think I am just using the same return codes as in standard SAT solvers.
> The returns code of evRouter are:
> -	0: the problem is unsatisfiable / unroutable
> -	1: the problem is routed and a solution is found
> -	2: the router cannot find a solution nor prove that it is unroutable with current limits.